### PR TITLE
#159790950 User should be redirected when cancel button is clicked

### PR DESCRIPTION
**Pull request by Orlayhemmy**
**What does this PR do?**
A user will be redirected when the cancel button on the add/edit asset category form is clicked

**Description of Task to be completed?**
 - call a method to redirect the user when cancel is clicked
 - write test

**How should this be manually tested?**
 - Navigate to the settings page
 - Click on the `Add new category` button
 - Click on the cancel button on the form displayed

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#159790950](https://www.pivotaltracker.com/story/show/159790950)

**Screenshots (if appropriate)**
N/A

**Questions:**
N/A

### CODE REVIEW

File: src/pages/CategoryPageForm/CategoryPageForm.test.tsx [hunk 1]
```diff
@@ -179,6 +179,14 @@ describe('The CategoryPageForm component', () => {
     });
   });
 
+  it('should redirect user when cancel button is clicked', () => {
+    const cancelButton = wrapper.find('.additional-fields__action-buttons').childAt(1);
+    cancelButton.simulate('click', {
+      preventDefault: jest.fn(),
```
Comment (andela-kakpobome) [inline src/pages/CategoryPageForm/CategoryPageForm.test.tsx]: Why's this here though?

Comment (Orlayhemmy) [reply src/pages/CategoryPageForm/CategoryPageForm.test.tsx]: Sorry I forgot to remove it. I did `event.preventDefault()` at first and when I removed it from the method, I forgot to effect it in the test.

**Comment by Orlayhemmy** [comment]
Thanks, Billmike, nice feedback. It has been implemented

File: src/pages/CategoryPageForm/CategoryPageForm.test.tsx [hunk 2]
```diff
@@ -179,6 +179,28 @@ describe('The CategoryPageForm component', () => {
     });
   });
 
+  it('modal title should be clear form on cancel button click', () => {
+    const cancelButton = wrapper.find('.additional-fields__action-buttons').childAt(1);
+    cancelButton.simulate('click');
+    expect(wrapper.find('.notifications-modal__heading').at(1).text()).toEqual('CLEAR FORM');
+  });
+
+  it('modal title should be clear form on cancel button click', () => {
```
Comment (andela-kakpobome) [inline src/pages/CategoryPageForm/CategoryPageForm.test.tsx]: How come 182 has the same title?

File: src/pages/CategoryPageForm/index.tsx [hunk 3]
```diff
@@ -393,6 +393,30 @@ export class CategoryPageForm extends React.Component<CategoryPageFormProps, Cat
...
+  cancelModal = () => {
+    return (
+      <NotificationModal
+        showModal = {this.state.isModalOpen}
+        handleCancel = {this.toggleModal}
+        action = "Yes"
+        cancelButtonTitle="No"
+        modalTitle = "CLEAR FORM"
+        modalContent = {{
+          heading: `Are you sure you want to cancel`,
```
Comment (andela-kakpobome) [inline src/pages/CategoryPageForm/index.tsx]: Add a question mark at the end

File: src/pages/CategoryPageForm/index.tsx [hunk 4]
```diff
@@ -393,6 +393,30 @@ export class CategoryPageForm extends React.Component<CategoryPageFormProps, Cat
...
 
+  /**
+   * This method redirects the user 
+   *
+   * @returns {JSX}
+   */
+  cancelModal = () => {
+    return (
+      <NotificationModal
+        showModal = {this.state.isModalOpen}
```
Comment (andela-kakpobome) [inline src/pages/CategoryPageForm/index.tsx]: Remove the space on the sides of the `=`

**Pull request closed without merging**